web/src/pages/Environments/EnvironmentContent.tsx

```diff
@@ -0,0 +1,31 @@
+import {useCallback, useState} from 'react';
+import SearchInput from '../../components/SearchInput';
+import {IEnvironment} from '../../redux/apis/TraceTest.api';
+import EnvironmentActions from './EnvironmentActions';
+import {EnvironmentState} from './EnvironmentState';
+import EnvironmentList from './EnvList';
+import * as S from './Envs.styled';
+import {EnvsModal} from './EnvsModal';
+
+const EnvironmentContent: React.FC = () => {
+  const [state, setState] = useState<EnvironmentState>({query: '', dialog: false, environment: undefined});
+  const onSearch = useCallback((value: string) => setState(st => ({...st, query: value})), [setState]);
+  const openDialog = useCallback((dialog: boolean) => setState(st => ({...st, dialog})), [setState]);
+  return (
+    <S.Wrapper>
+      <S.TitleText>All Envs</S.TitleText>
+      <S.PageHeader>
+        <SearchInput onSearch={onSearch} placeholder="Search test" />
+        <EnvironmentActions openDialog={() => openDialog(true)} />
```

Comment:
Open dialog doesn't provide much info, what dialog are we opening and for what reason?

Reply:
i am passing the Dispatch function from the useState call called setIsFormOpen ✅